e2e/entityid/generator.go

```diff
@@ -0,0 +1,48 @@
+package entityid
+
+import (
+	"math/rand"
+	"regexp"
+	"strconv"
+	"time"
+)
+
+const (
+	// idChars represents the character set used to generate cluster IDs.
+	// (does not contain 1 and l, to avoid confusion)
+	idChars = "023456789abcdefghijkmnopqrstuvwxyz"
+
+	// idLength represents the number of characters used to create a cluster ID.
+	idLength = 5
+)
+
+var (
+	// Use local instance of RNG. Can be overwritten with fixed seed in tests
+	// if needed.
+	localRng = rand.New(rand.NewSource(time.Now().UnixNano()))
+)
+
+func New() string {
```

Comment:
New sounds to me like I'm building an object. I'd prefer `NewID`.

Reply:
In that case you would invoke it with `entityid.NewID()`, which AFAIU is against golang philosophy of not repeating parts of the name :)

Does `entity.NewID()` sound better maybe?

Reply:
`entityid.Generate()`?